src/en/mangago/src/eu/kanade/tachiyomi/extension/en/mangago/Mangago.kt

```diff
@@ -170,14 +171,18 @@ class Mangago : ParsedHttpSource(), ConfigurableSource {
 
     override fun searchMangaNextPageSelector() = genreListingNextPageSelector
 
-    private val titleRegex = Regex("""\(yaoi\)|\{Official\}|«Official»|〘Official〙|\(Official\)|\s\[Official]|\s「Official」|『Official』|\s?/Official\b""", RegexOption.IGNORE_CASE)
-    private fun titleVersion(title: String) = title.replace(titleRegex, "").trim()
+    private var titleRegex: Regex =
+        Regex(
+            "(?:\\([^()]*\\)|\\{[^{}]*\\}|\\[(?:(?!]).)*]|«[^»]*»|〘[^〙]*〙|「[^」]*」|『[^』]*』|≪[^≫]*≫|﹛[^﹜]*﹜|𖤍.+?𖤍|/.+?)\\s*|([|/~].*)",
+            RegexOption.IGNORE_CASE,
+        )
 
```

Comment:
This is too wide, if you want to remove just the Official or yaoi then add it into the list.

Reply:
Actually all of those symbols are present in titles in mangago as well
it can be (Official) or «Official» for example
It only depends on the person creating the title
they could be using any of those

Reply:
btw it could also have any other kind of text like Side Story or MGS or anything like that

Reply:
Do I need to change anything else in the mangago/batoto PRs?

Reply:
I think what Keiyoushi said makes sense, if you found a new symbol that should be stripped from the title, just create an update for everyone to use it.